sqlitecluster/SQLiteNode.cpp

```diff
@@ -458,6 +458,22 @@ list<STable> SQLiteNode::getPeerInfo() const {
     return peerData;
 }
 
+string SQLiteNode::getEligibleFollowerForForwarding() const {
+    vector<string> validPeers;
+    const string leaderVersion = getLeaderVersion();
```

Comment:
We should return the empty string if leaderVersion is the empty string.

Reply:
Done